website/content/en/docs/building-operators/golang/migration.md

```diff
@@ -1,14 +1,13 @@
 ---
-link: Migrating pre-v1.0.0 Projects
-linkTitle: Migrating pre-v1.0.0 Projects
+link: Migrating Projects from pre-v1.0.0 to the latest release
+linkTitle: Migrating from pre-v1.0.0 to latest
 weight: 200
-description: Instructions for migrating a Go-based project built prior to v1.0.0 to use the new Kubebuilder-style layout.
+description: Instructions for migrating a Go-based project built prior to `v1.0.0` (`0.19.x+`) to use the Kubebuilder-style layout which is the default layout adopted by SDK since the `1.0.0` release.
 ---
 
 ## Overview
 
-The motivations for the new layout are related to bringing more flexibility to users and
-part of the process to [integrate Kubebuilder and Operator SDK][integration-doc].
+The motivation for the new layout was to [integrate Kubebuilder and Operator SDK][integration-doc] based projects. This enables Kubebuilder based projects to use the features provided by Operator SDK. For further information behind this motivation, check [What are the the differences between Kubebuilder and Operator-SDK?][what-are-the-the-differences-between-kubebuilder-and-operator-sdk]. 
```

Comment:
Nit: This may look better
```suggestion
The motivation for the new layout was to [integrate Kubebuilder and Operator SDK][integration-doc] based projects. This enables Kubebuilder based projects to use the features provided by Operator SDK. For further information behind this motivation, check [the differences between Kubebuilder and Operator-SDK][what-are-the-the-differences-between-kubebuilder-and-operator-sdk]. 
```